### ISSUE TITLE
NanoPi2 - since v2.0.0 no display output or network connect anymore

### ISSUE THREAD

**Issue by Marsupilani** [WLAN-Pi/releases#25]
on a nanopi2 I didn't have any problems until v1.9.1, but since v2.0.0 it seems the image does not work anymore. I tested several micro sdcard, which all worked until v1.9.1, but not with v2.0.0. I do not get any display output anymore and the IP169.254.42.1 is not reachable,too...

**Comment by jolla** [comment]
If you re-image with v1.9.1 does it still work? Have you tried re-downloading the v2 image?

**Comment by Marsupilani** [comment]
Yes, with V1.9.1 everything work. I downloaded the V2 image several times & flashed it to different cards and different tools,too (Rufus/etchr), everytime the same - the v2 doesn't work at all. No display and no network connectivity. The v1.9.1 version worked with the same device and card.

**Comment by KB8DOA** [comment]
Noticed the same issue today when I applied v2 to my PiNeo.
Seems like something gets damaged with the display driver if you power it off without shutting down first.
After re-flashing v2, the display came back on mine.
It does take quite some time for the display to initalize, thus could confuse user into into thinking that it re-flash of v2 did not work.
Am now investigating why the Gigabit speedtest is so terrible on mine now.
Gigabit testing showing <100 Meg most of the time, and when not - only showing about 300-400 Meg.
I am downgrading to older versions to try to see if I can get reliable speedtesting again... I will post again once I figure this out.

**Comment by kaybeng** [comment]
I just got a Neo2 with 512MB ram. The device powers up with the supplied Ubuntu-oled sdcard and I was able to connect to it via Ethernet. I flashed v2 or v1.9.1 to different sdcards and am unable to get it to work. The screen is blank and both ethernet ports are always lighted. May I have some suggestions to diagnose the issue.

**Comment by jolla** [comment]
@Marsupilani the WLAN Pi image is only compatible with the NanoPi **NEO2** boards, above you mentioned "nanopi2", Please confirm you are using a NEO2 board.

**Comment by credmond06** [comment]
Running into similar issue. Using NEO2 and v2.0.1, will not boot with the wifi adapter plugged in. I remove the adapter and reboot, it boots up. I plug the adapter back in, and menus are frozen, (I cannot select any menu etc) I re-flash to v1.9.1, and no issues. I re-flash to v2.0.1, and still will not boot with wifi adapter plugged in. If I navigate to Network adapter > WLAN interfaces, and plug the adapter in, it work. However, when I try to boot to profiler, or Hotspot, it gets stuck. If I boot to those modes without the wifi adapter in it, it works. It almost seems like the NEO2 is seeing the adapter as a boot device.

**Comment by jolla** [comment]
Which Wi-Fi adapter are you using?

**Comment by credmond06** [comment]
Comfast cf-912ac

**Comment by credmond06** [comment]
Anyone else having this issue?

**Comment by joshschmelzle** [comment]
@credmond06 when FPMS is frozen can you gather and paste in the output of `sudo service fpms status`?

**Comment by credmond06** [comment]
So, TOTAL NOOB job here. When I went to flash v2.0 onto the SD card, it kept failing validation in Etcher. I then realized I was trying to load the zip'd download file .img.gz. I unzipped the file to a .img and re-flashed the SD card. Works perfectly.

**Comment by jolla** [comment]
Glad to hear you got it working. Etcher actually should be able to take the zip directly, but perhaps you are running an older version of Etcher?

**Comment by Janwillem26** [comment]
Had the same issues as Marsipulani described initially. v1.9.1 working fine on my NEO2. No joy with v2.0.1 (also tried v2.0.0). Various etch attempts different cards no joy. Plug in the old v1.9.1... boots in seconds.

With the new image, the red LED would come on with power and light up more intense after a few seconds with the green LED starting to flash for a couple of times and after that no joy.

Reading through the NEO2 specs I noticed the power requirements. As my power source was a left over USB-block I checked its output rating. Too low. Changed it for another bigger block and guess what...... Joy!

Guess v2.0.1 emits more CO2 then.
